example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
+    # to participate.
+    minimum_participant_icp:     10 tokens
+
+    # The maximum amount of ICP that each participant must contribute
+    # to participate.
+    maximum_participant_icp: 10_000 tokens
+
+    # The optional text that swap participants must confirm before they may
+    # participate in the swap. If set, must be within 1 to 1,000 characters.
     # confirmation_text: >
-    #     Do you understand the responsibilities and risks associated
-    #     with participating in this token swap?
+    #     I confirm my understanding of the responsibilities and risks
+    #     associated with participating in this token swap.
 
+    # An optional list of countries that should not participate in the
+    # decentralization swap. By default, participants from all countries
+    # are allowed.
     restricted_countries:
         - AQ  # Antarctica
 
+    # Configuration of the vesting schedule of the neuron basket, i.e., the SNS
+    # neurons that a participants will receive from a successful
+    # decentralization swap.
     VestingSchedule:
-        events: 104
-        interval: 7 d
+        # The number of events in the vesting schedule. This translates to how
+        # many neurons will be in each participant's neuron basket.
+        events: 3
+
+        # The interval at which the schedule will be increased per event. The
+        # first neuron in the basket will be unlocked with zero dissolve delay.
+        # Each other neuron in the schedule will have its dissolve delay
+        # increased by `interval`.
+        interval: 1 month
 
-    start_time: 12:00 UTC # noon
+    # Absolute time of day when the decentralization swap is supposed to start.
+    #
+    # An algorithm will be applied to allow at least 24 hours between the time
+    # of execution of the CreateServiceNervousSystem proposal and swap start.
+    # For example, if start_time is 23:30 UTC and the proposal is adopted and
+    # executed at 23:20 UTC, then the swap start will be at 23:30 UTC the next
+    # day (i.e., in 24 hours and 10 min from the proposal execution time).
+    #
+    # WARNING: Swap start_time works differently on mainnet and in testing.
+    #
+    # On mainnet:
+    # - Setting start_time to some value (e.g., 23:30 UTC) will allow the swap
+    #   participants to be prepared for the swap in advance, e.g.,
+    #   by obtaining ICPs that they would like to participate with.
+    # - If start_time is not specified, the actual start time of the swap will
+    #   be chosen at random (allowing at least 24 hours and less than 48 hours,
+    #   as described above).
+    #
+    # In testing:
+    # - Setting start_time to some value works the same as explained above.
+    # - If start_time is not specified, the swap will begin immediately after
+    #   the CreateServiceNervousSystem proposal is executed. This facilitates
+    #   testing in an accelerated manner.
+    #
+    # start_time: 23:30 UTC
```

Comment:
is this an old example and should be removed?

Reply:
```suggestion
    # start_time: 23:30 UTC  # Intentionally commented out for testings
```